addons/mass_mailing/data/mailing_data_templates.xml

```diff
@@ -7,14 +7,27 @@
     -->
     <template id="mass_mailing_mail_layout">
         &lt;!DOCTYPE html&gt;
-        <html xmlns="http://www.w3.org/1999/xhtml">
+        <html xmlns="http://www.w3.org/1999/xhtml" xmlns:v="urn:schemas-microsoft-com:vml" xmlns:o="urn:schemas-microsoft-com:office:office">
             <head>
                 <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
                 <meta name="format-detection" content="telephone=no"/>
                 <meta name="viewport" content="width=device-width; initial-scale=1.0; maximum-scale=1.0; user-scalable=no;"/>
                 <meta http-equiv="X-UA-Compatible" content="IE=9; IE=8; IE=7; IE=EDGE" />
 
                 <t t-call="mass_mailing.mass_mailing_mail_style"/>
+
+                <!--
+                    Prevent Outlook from distorting images with DPI scaling (see
+                    https://www.htmlemailcheck.com/knowledge-base/dpi-scaling-in-outlook-2007-2013/):
+                -->
+                <!--[if mso]>
+                    <xml>
+                        <o:OfficeDocumentSettings>
+                            <o:AllowPNG/>
+                            <o:PixelsPerInch>96</o:PixelsPerInch>
+                        </o:OfficeDocumentSettings>
+                    </xml>
+                <![endif]-->
```

Comment:
Note for the support team: I know this looks weird at first glance, but this particular HTML comment is _not_ a oversight. This is an actual feature and this comment _must_ be applied in the customer's database alongside the other xml changes in this file. Comments which starts with `[if mso]` are actually interpreted by some versions of Microsoft Outlook and that trick is used here to provide rendering directives that will only affect those particular email clients without affecting the others.